pycon/static/less/bootstrap/mixins.less

```diff
@@ -558,13 +558,13 @@
   .core (@gridColumnWidth, @gridGutterWidth) {
```

Comment:
lifted from https://gist.github.com/metaskills/5024393. It seems there were some changes from the previous lessc version (1.3.3) to the version available from Ubuntu 14.04 (1.4.2)